docker-compose.yml

```diff
@@ -1,5 +1,35 @@
 version: '3'
 services:
+  ssm-postgres:
+    image: postgres
+    restart: unless-stopped
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: postgres
+
+  ssm-keycloak:
+    image: keycloak
+    restart: unless-stopped
+    depends_on:
+      - ssm-postgres
+    ports:
+      - "8080:8080"
+    environment:
+      FEDERATE_LDAP: "true"
+      GOOGLE_CLIENT_ID: 308694469630-7l6odifs3dc88arbh5cia6ud7v4h2suo.apps.googleusercontent.com
+      GOOGLE_IDP: "true"
+      GOOGLE_SECRET: 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
```

Comment:
To check, are these actual secrets that need to be removed?

Just wanna make sure since we can remove them in the feature branch probably easier than in the `main` and `next` branches.

Reply:
Hi Robert, just checking, can we remove these secrets?

Reply:
@marshallmcdonnell Yes, but these aren't real secrets. They're the credentials to a Google app that exists only to test SSM's authentication with.

Reply:
Thanks! I'll put in "XXXX..." fillers. **Sound good?** I'd rather not have things that look like "real" secrets (may get picked up via automated GitHub security scanning)